### PR TITLE
Refactoring Shared-Verify-Tools workflow to output artifacts names

### PR DESCRIPTION
**Pull request by lemraus**

### CODE REVIEW

**Comment by marcadetd** [comment]
For variable names (inputs and outputs), use either snake_case or camelCase, but not both

**Comment by lemraus** [comment]
Those changes are now ready for review. The added output parameters work as expected, as you can see in the CI runs on [this PR](https://github.com/riseclipse/riseclipse-validator-scl2003/pull/69).

**Review by SoaAlex — APPROVED** [review]

**Comment by marcadetd** [comment]
naming is still mixed: artifact1Name and artifact_1_name

**Comment by lemraus** [comment]
> naming is still mixed: artifact1Name and artifact_1_name. I prefer the latter.

@marcadetd The issue is that the `jarPath1` input is referenced in a lot of workflows, which means that changing the case now would require to generate many more PRs (one per repo). Do you prefer to use camel case on all inputs/outputs in this file, or change for snake case in this file and all workflows that consume this reusable workflow?

File: .github/workflows/Shared-Verify-Tools.yml [hunk 1]
```diff
@@ -31,73 +31,86 @@ on:
...
 
-    - name: Upload artifact 1
-      uses: actions/upload-artifact@v3
-      with:
-        name: ${{ steps.artifact_name.outputs.artifact_1_filename }}-${{ steps.get_version.outputs.project_version }}.jar
-        path: ${{ github.workspace }}/${{ steps.artifact_name.outputs.artifact_1_no_ext }}-${{ steps.get_version.outputs.project_version }}.jar
+      # This step is tricky but allows to keep the version in the uploaded artifacts
+      - name: Get artifact name et basedir from jarPath
+        run: |
+          jarPath1="${{ inputs.jarPath1 }}"
```
[the remark this answers is not in the document]
Comment (lemraus) [reply .github/workflows/Shared-Verify-Tools.yml:90]: Will change the name to `JAR_PATH_1`, indeed the former version does not look like a suitable environment variable name.

[the remark this answers is not in the document]
Comment (marcadetd) [reply .github/workflows/Shared-Verify-Tools.yml:90]: But JAR_PATH_1 is not in line with the other environment variable names, that's why I suggest ARTIFACT_1_JAR

**Comment by lemraus** [comment]
> Just two PR: riseclipse-developer and riseclipse-validator-scl2003. I prefer snake_case for all inputs/outputs

Perfect then I will take care of it this afternoon, I thought this workflow was used by more repos but looks like it is not the case.

**Comment by lemraus** [comment]
@marcadetd Changes should now be complete for both repos regarding variable names.

**Review by marcadetd — APPROVED** [review]

**Comment by marcadetd** [comment]
OK to merge

**Pull request merged**